util/test-utils/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+#![allow(warnings)]
+use tempfile::{NamedTempFile, TempDir};
+
+/// Create a temporary directory in the directory specified by the
+/// cargo-provided `OUT_DIR` environment variable.
+///
+/// # Panics
+///
+/// - If `OUT_DIR` doesn't exist
+/// - If [`TempDir::new_in`] fails to create the directory.
+pub fn tempdir() -> TempDir {
```

Comment:
I think we'll want a variant that takes a prefix, e.g. `tempdir_with_prefix()`. It can use [Builder::prefix()](https://docs.rs/tempfile/latest/tempfile/struct.Builder.html#method.prefix)